### ISSUE TITLE
do we run into Groq rate limits

### ISSUE THREAD

**Issue by kjappelbaum** [lamalab-org/matextract-book#188]
I think I ran into some when using the `choosing_paradigm` notebook now. Do you think we should add a note or when a sleep, @MrtinoRG?

**Comment by MrtinoRG** [comment]
Yes, I think they considerably decreased the rate limits for the free plan, so yeah it probably needs some sleeps
